src/components/menu/ComingSoon/ComingSoon.js

```diff
@@ -0,0 +1,44 @@
+import './GoogleBrandFormatting.css'
+
+
+function ComingSoon ()
+{
+    return ( 
+        <div className='center'>  
+<h1 className='h1-green'> <span class = "h1-blue">
```

Comment:
I'm surprised this worked, but in React you would use `className` instead of `class`
```suggestion
<h1 className='h1-green'> <span className = "h1-blue">
```